src/api/app/controllers/webui/request_controller.rb

```diff
@@ -15,6 +15,7 @@ def show
     if Flipper.enabled?(:request_show_redesign, User.session)
       @is_author = @bs_request.creator == User.possibly_nobody.login
       @is_target_maintainer = @bs_request.is_target_maintainer?(User.session)
+      @can_handle_request = BsRequestPolicy.new(User, @bs_request).can_handle_request?(@is_target_maintainer, @is_author)
```

Comment:
As we discussed together @ncounter and I, this authorization check could be inlined in the view (with the usual `policy(my_record).my_method?`) instead of being passed around through multiple view levels with the `@can_handle_request` variable.

Reply:
Thanks a lot for all the tips and the whole overview about policies and Pundit.
I addressed them all now.